frontend/src/components/searchBar.component.js

```diff
@@ -66,17 +65,17 @@ class SearchBar extends React.Component {
     }
 
     //These functions handle the min price and max price
-    handleMinPrice = (event) => {
-        this.setState({
-            minPrice: event.target.value
-        })
-    }
-
-    handleMaxPrice = (event) => {
-        this.setState({
-            maxPrice: event.target.value
-        })
-    }
+    // handleMinPrice = (event) => {
+    //     this.setState({
+    //         minPrice: event.target.value
+    //     })
+    // }
+    //
```

Comment:
Same with this one?

Reply:
same with this, can be uncommented later on

Reply:
sounds good! I will approve it!